ouroboros-consensus-test/test-storage/Test/Ouroboros/Storage/ChainDB/Model.hs

```diff
@@ -621,6 +634,7 @@ class ( HasHeader blk
 -------------------------------------------------------------------------------}
 
 type InvalidBlocks blk = Map (HeaderHash blk) (InvalidBlockReason blk, SlotNo)
+type ValidBlocks blk = Set (HeaderHash blk)
```

Comment:
As far as I can see we use `ValidBlocks` only once. Would it make sense to remove the extra level of indirection?